letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -96,6 +96,10 @@ elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-njalla" ]; then
     bashio::config.require 'dns.njalla_token'
     PROVIDER_ARGUMENTS+=("--authenticator" "certbot-dns-njalla:dns-njalla" "--certbot-dns-njalla:dns-njalla-credentials" /data/dnsapikey "--certbot-dns-njalla:dns-njalla-propagation-seconds" "${PROPAGATION_SECONDS}")
 
+# rfc2136
+elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "rfc2136" ]; then
```

Comment:
```suggestion
elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-rfc2136" ]; then
```
`rfc2136` isn't an option in here:
https://github.com/home-assistant/addons/blob/d37b21354316c7266a5d7616a553bfe15da5af71/letsencrypt/config.yaml#L72-L76

Should be `dns-rfc2136`